src/app/components/media/MediaOriginBanner.js

```diff
@@ -109,11 +109,11 @@ const MediaOriginBanner = ({
   projectMedia: {
     media_cluster_origin: origin,
     media_cluster_origin_timestamp: originTimestamp,
-    media_cluster_origin_user: { name: user } = {},
+    media_cluster_origin_user: user = {},
     media_cluster_relationship: mediaClusterRelationship,
   },
 }) => {
-  const { icon, message } = getIconAndMessage(origin, mediaClusterRelationship, user, originTimestamp);
+  const { icon, message } = getIconAndMessage(origin, mediaClusterRelationship, user?.name, originTimestamp);
```

Comment:
[comment] (non-blocker) If user is being assigned an empty object by default then the safe navigation operator shouldn't be necessary, right?

Reply:
removed the unnecessary safe operator, @amoedoamorim :saluting_face: